tools/watch_logs.py

```diff
@@ -9,7 +9,59 @@
 import io
 import time
 import threading
-from baselayer.log import colorize
+import logging
+import socket
+from logging.handlers import SysLogHandler
+
+from baselayer.log import (colorize, make_log)
+from baselayer.app.env import load_env
+
+env, cfg = load_env()
+log = make_log("watch_logs")
+
+
+def check_external_logging():
+    """
+    Check 3rd party logging, if enabled, and make sure that
+    it is set up properly
+
+    TODO: This could eventually be done with a JSONschema
+    """
+    enabled_services = []
+    external_logging_enabled = False
+
+    if not cfg.get('external_logging'):
+        return external_logging_enabled, enabled_services
+
+    external_logging_enabled = cfg['external_logging']["enabled"]
+    if not external_logging_enabled:
+        return external_logging_enabled, enabled_services
+
+    print(cfg['external_logging'])
+    for service, config in cfg['external_logging']["services"].items():
```

Comment:
Should this become a separate microservice rather?  We cannot guarantee that watch_logs.py will be run otherwise.

That allows us to also use the existing enable/disable microservice configuration.

Reply:
Interesting idea. @dannygoldstein what do you think?

Reply:
sounds good to me